apps/andi/lib/andi_web/live/dataset_live_view/table.ex

```diff
@@ -44,4 +44,7 @@ defmodule AndiWeb.DatasetLiveView.Table do
 
   defp get_status_class(nil), do: "unset"
   defp get_status_class(status), do: String.downcase(status)
+
+  defp edit_type(false), do: "submissions"
```

Comment:
I just shotgunned this in as an implementation. Let me know if you'd like it to suck less